src/main/webapp/app/shared/pipes/artemis-date.pipe.ts

```diff
@@ -144,7 +144,7 @@ export class ArtemisDatePipe implements PipeTransform, OnDestroy {
     }
 
     private cleanUpSubscription(): void {
-        if (this.onLangChange != undefined) {
+        if (this.onLangChange !== undefined) {
```

Comment:
Can you revert this change please? (Just using `!=` here checks for both undefined and null, so is preferred.)